nixos/modules/tasks/swraid.nix

```diff
@@ -1,47 +1,71 @@
 { config, pkgs, lib, ... }: let
 
-  cfg = config.boot.initrd.services.swraid;
+  cfg = config.boot.swraid;
 
 in {
 
-  options.boot.initrd.services.swraid = {
+  options.boot.swraid = {
```

Comment:
We missed adding a mkRenamedOption for this

```
       error: attribute 'swraid' missing

       at /nix/store/6vrnicwhzppfy58v8l8jhpjplzckd74k-source/configuration.nix:103:78:

          102|       # https://github.com/numtide/srvos/blob/master/common/default.nix#L20
          103|       systemd.enable = lib.mkDefault (!config.boot.initrd.network.enable && !config.boot.initrd.services.swraid.enable && !config.boot.isContainer);
             |                                                                              ^
          104|     };
```

Reply:
Opened #243577